src/main/java/com/greghaskins/spectrum/Box.java

```diff
@@ -0,0 +1,38 @@
+package com.greghaskins.spectrum;
+
+/**
+ * For use with nested steps inside a test, this box allows data passing between tests
+ */
+public class Box<T> {
```

Comment:
I could rename it Value if you like. I see it used to be hiding inside the Spectrum class and used to have public field access :(

The `let()` construct is a nice idea, but is intended to stop leaking between specs. In a BDD context, the leaf nodes aren't specs, they're steps, and as such should be able to communicate. Boxed values would like us do that.

Tell me what to change to make you approve (other than the failing javadoc :))